src/wallet/wallet.cpp

```diff
@@ -781,7 +781,7 @@ bool CWallet::IsSpent(const COutPoint& outpoint) const
     return false;
 }
 
-void CWallet::AddToSpends(const COutPoint& outpoint, const uint256& wtxid, WalletBatch* batch)
+void CWallet::AddToSpends(const COutPoint& outpoint, const Txid& wtxid, WalletBatch* batch)
```

Comment:
nit: would be a nice opportunity to fix the confusing naming and just call this `txid`